internal/services/compute/orchestrated_virtual_machine_scale_set_resource_test.go

```diff
@@ -414,6 +415,50 @@ func TestAccOrchestratedVirtualMachineScaleSet_importLinux(t *testing.T) {
 	})
 }
 
+func TestAccOrchestratedVirtualMachineScaleSet_skuProfileWithoutSkuName(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_orchestrated_virtual_machine_scale_set", "test")
+	r := OrchestratedVirtualMachineScaleSetResource{}
+
+	// currently only supported in EastUS, WestUS, EastUS2, and WestUS2
+	data.Locations.Primary = "eastus2"
```

Comment:
Can we update this to be a TODO so we know it needs to be removed when the Preview is rolled out to our normal testing region(s).

Can we get an estimate from the Service team as to when this will be deployed to the normal regions?

```suggestion
	// TODO - Remove this override when Preview is rolled out to westeurope - currently only supported in EastUS, WestUS, EastUS2, and WestUS2
	data.Locations.Primary = "eastus2"
```

Reply:
the current rollout which will take approximately 2 weeks will enable the feature in all regions